Dockerfile

```
@@ -0,0 +1,5 @@
+FROM alpine:3.10
+RUN mkdir .airplane
+COPY ./airplane .airplane
+ENV PATH=".airplane:${PATH}"
+ENTRYPOINT ["airplane"]
```

```suggestion
FROM alpine:3.10

RUN mkdir -p /airplane/bin
COPY ./airplane /airplane/bin/airplane
ENV PATH="/airplane/bin:${PATH}"

ENTRYPOINT ["/airplane/bin/airplane"]
```

- i think you can just use a root directory here, don't rely on current user or current directory
- ENTRYPOINT to absolute path, again don't rely on current directory